File: LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentTokenizeCallback.java
package com.braintreepayments.api;

import android.content.Context;

import androidx.annotation.NonNull;

/**
 * Callback for receiving result of
 * {@link LocalPaymentClient#tokenize(Context, LocalPaymentAuthResult, LocalPaymentTokenizeCallback)}.
 */
public interface LocalPaymentTokenizeCallback {

    /**
     *
     * @param localPaymentResult a success, failure, or cancel result from the local payment flow
     */
    void onLocalPaymentResult(@NonNull LocalPaymentResult localPaymentResult);
}

File: LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentTokenizeCallback.kt
package com.braintreepayments.api

/**
 * Callback for receiving result of
 * [LocalPaymentClient.tokenize].
 */
fun interface LocalPaymentTokenizeCallback {
    /**
     *
     * @param localPaymentResult a success, failure, or cancel result from the local payment flow
     */
    fun onLocalPaymentResult(localPaymentResult: LocalPaymentResult)
}
